### PR TITLE
Switch to numerical Yubikey public IDs

### PR DESCRIPTION
**Pull request by rjkip**
- [x] Switch StepupBundle back to `dev-develop`

### CODE REVIEW

**Comment by rjkip** [comment]
:mega: @DRvanR

File: src/Surfnet/StepupMiddleware/MiddlewareBundle/Service/EventStreamReplayer.php [hunk 1]
```diff
@@ -142,7 +142,10 @@ public function replayEvents(OutputInterface $output, $increments)
             }
 
             $this->connectionHelper->commit();
-            $replayProgress->finish();
+
+            if ($replayProgress->getProgress() > 0) {
```
Comment (DRvanR) [inline src/Surfnet/StepupMiddleware/MiddlewareBundle/Service/EventStreamReplayer.php]: eh?

**Comment by DRvanR** [comment]
One small question, bar that :+1:

File: src/Surfnet/StepupMiddleware/MiddlewareBundle/Service/EventStreamReplayer.php (hunk 1 above)
Comment (rjkip) [inline src/Surfnet/StepupMiddleware/MiddlewareBundle/Service/EventStreamReplayer.php]: This allows replay of an empty event stream.

**Comment by DRvanR** [comment]
:+1: pending CIs

**Pull request merged**